pip/flatpak-pip-generator

```diff
@@ -114,9 +120,21 @@ for package in packages:
             module['cleanup'] = ['/bin', '/share/man/man1']
 
         try:
-            subprocess.run(pip_download + [
-                '--no-binary', ':all:', package
-            ], check=True)
+            sdist_package = None
+            subprocess.run(pip_download + [package], check=True)
+            for filename in os.listdir(tempdir):
+                if filename.rsplit('.', 1)[1] != 'gz' and filename.rsplit('-', 1)[1] != 'any.whl':
+                    cwd = os.getcwd()
```

Comment:
Whats with the directory dance?

Reply:
Thanks for the feedback, I agree this wasn't very elegant. It is now using `os.path.join` instead of changing the directory and then changing it back.